pkg/providers/docker/config/template-md.yaml

```diff
@@ -9,6 +9,18 @@ spec:
       joinConfiguration:
         nodeRegistration:
           criSocket: /var/run/containerd/containerd.sock
+{{- if .workerNodeGroupTaints }}
+          taints: {{ range .workerNodeGroupTaints}}
+            - key: {{ .Key }}
+              value: {{ .Value }}
+              effect: {{ .Effect }}
+{{- if .TimeAdded }}
+              timeAdded: {{ .TimeAdded }}
+{{- end }}
+          {{- end }}
```

Comment:
```suggestion
{{- end }}
```

Reply:
It is the end of the range loop, hence indented accordingly.

Reply:
Can we move the range statement to its own line then? And then that way we won't have indented blocks which will make it uniform to everything else (or if we keep indented it will be more clear which one it is referring to)